server/routes/creatingLicences/handlers/initialMeetingTime.ts

```diff
@@ -1,15 +1,20 @@
 import { Request, Response } from 'express'
+import LicenceService from '../../../services/licenceService'
+import { jsonToSimpleDateTime } from '../../../utils/utils'
 
 export default class InitialMeetingTimeRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
   GET = async (req: Request, res: Response): Promise<void> => {
-    const offender = {
-      name: 'Adam Balasaravika',
-    }
-    res.render('pages/create/initialMeetingTime', { offender })
+    const { appointmentTime } = res.locals.licence
+    const formDate = jsonToSimpleDateTime(appointmentTime)
+    res.render('pages/create/initialMeetingTime', { formDate })
   }
```

Comment:
I wondered about this bit. An alternative way would be to use the `fetchLicenceMiddleware` to also do the appointment date and time conversion to a SimpleDateTime and add it into the licence object to simplify the route code and make all date/time conversions done in the service layer. Discuss.

Reply:
I think what you have done is fine in general. I'm not a fan of converting it in the middleware, because it might be useful elsewhere to have it in a format that can be imported to moment for date calculations. The only use case for the SimpleDateTime format is to split it out into the 6 input fields we have on our forms